gitkraken-client/experimental-features.md

```diff
@@ -0,0 +1,79 @@
+
+---
+
+Coming from the icy depths of the git ocean are GitKraken Client experimental features. These are ideas we are trying out that still have alot of work ahead, but that we want to share with the world sooner. 
+
+---
+
+## Experimental Features
+
+Navigate to <i class="fas fa-cog"></i><kbd><strong>Preferences</strong><i class='fa fa-caret-right'></i><strong>Experimental</strong></i></kbd> to access the experimental features menu.
+
+Experimental Features are still under development - treat these as an early sneak peek at some of the new functionality we're working on at GitKraken. Experimental features may not work as intended and could be changed or removed in the future. These settings are entirely optional and can be turned off at any time.
+
+If you do experience issues, or have any other feedback, please reach out to us [Contact Support](https://help.gitkraken.com/gitkraken-client/contact-support/?issue_category__customer_facing_field_=Experimental+feedback&subject=GitKraken+Client+Experimental+feedback).
+
+### Git Executable
+
+When this setting is enabled, GitKraken will utilize the Git executable instead of the NodeGit library for certain Git actions including fetching and committing. This may provide increased performance and compatibility with certain projects and development environments. This is a partial implentation and will only effect some aspects of Git within GitKraken - the amount of git commands using the git executable will increase with each release.
+
+**Use Git Executable:** ☑ turn on this experimental feature! As soon as you check the box it is working.
+
+**Git Executable:** We automatically include git with GitKraken. You can select other from other versions that are installed on your system. 
+
+##### Features using the Git Executable
+
+Find below a list of features using the Git binary if the Git Executable experimental setting is enabled. Anything not listed is still using LibGit2/NodeGit paths.
+
+**Added in 9.4.0:**
+- refresh commits (some of the commit info displayed in the graph)
+- verify commit gpg signature
+- commit
+- fetch
+- branch ahead/behind count (e.g. from pull request panel)
+- merge base calculation (e.g. from right clicking a local branch in the left panel)
+- branch rename
+- branch delete (local branch only)
+
+### AI Commit Message Generation
+
+With this setting enabled, GitKraken will have the ability to generate commit messages from any staged changes. You can access this feature from the Command Palette (⌘P/Ctrl P, or the magic wand icon () in the toolbar), or you can click the sparkle icon () in the commit panel when you have staged changes.
+
+<div class='callout callout--danger'>
+    <p> This feature requires sending a diff of your code changes to OpenAI. Ensure your code does not contain sensitive information before generating a commit message.</p>
+</div>
+
+**Generate Commit Messages with AI:** ☑ turn on this experimental feature! As soon as you check the box it is working.
```

Comment:
Change `As soon as you check the box it is working.` to `Check the box to immediately enable the setting.`